src/SourceBuild/content/eng/tools/BinaryToolKit/BinaryToolKit.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" InitialTargets="CheckRuntimeVersion">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <RuntimeVersion></RuntimeVersion>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <Target Name="CheckRuntimeVersion">
+    <Error Condition="'$(RuntimeVersion)' == ''" Text="RuntimeVersion is not set. Please consider passing as an MSBuild Property" />
+  </Target>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="$(RuntimeVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="$(RuntimeVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="$(RuntimeVersion)" />
+    <PackageReference Include="System.CommandLine" Version="2.0.0-*" />
```

Comment:
Hmm, this looks weird, why do we need a floating version? And shouldn't the version be defined in Versions.props?

Reply:
Good point. I've modified this to use a non-floating version, and I've migrated the versions to a Versions.props file. 
Unsure if having $(RuntimeVersion) in the props file like I've done in https://github.com/dotnet/installer/pull/18726/commits/80f9f80a9eecccfb644a479d038ffbc729bfe27d is correct or if the